website/src/pages/roadmap.md

```diff
@@ -10,13 +10,13 @@ down by areas on our [stack](blog/2021/07/21/streaming-data-lake-platform/#hudi-
 
 ## H1 2022 Releases
 
-Next minor release : [0.10.1](https://issues.apache.org/jira/projects/HUDI/versions/12351135) (Jan 2022)
-Next major release : [0.11.0](https://issues.apache.org/jira/projects/HUDI/versions/12350673) (Feb 2022)
+Next minor release : [~~0.10.1~~](https://issues.apache.org/jira/projects/HUDI/versions/12351135) ~~(Jan 2022)~~
```

Comment:
as discussed, we can remove already released version 0.10.1 from this page

Reply:
Done. Please check.